api/camunda/camunda-openapi.yaml

```diff
@@ -1012,11 +1501,56 @@ components:
           format: int64
         tenantId:
           type: string
-    CamundaUserWithPasswordRequest:
+    AuthorizationAssignRequest:
+      type: object
+      properties:
+        ownerKey:
+          type: string
+        ownerType:
+          type: string
+          enum:
+            - USER
+            - ROLE
+            - GROUP
+        resourceKey:
+          type: string
+        resourceType:
+          type: string
+        permissions:
+          type: array
+          items:
+            type: string
```

Comment:
change-at-the-source, descriptions, new:

These properties all need descriptions. We should follow up with the original author of this endpoint to add them.

Reply:
@Ben-Sheppard - in [a recent PR](https://github.com/camunda/camunda/pull/20963) to the C8 REST API, you added the following, all of which are missing `description` properties in the OpenAPI spec:

- [POST /authorizations endpoint (now the PATCH /authorizations/{ownerKey} endpoint](https://github.com/camunda/camunda/blob/fc1cac538558a1a9b510888060b957bdba0f961b/zeebe/gateway-protocol/src/main/proto/rest-api.yaml#L841) endpoint 
- [AuthorizationAssignRequest (now named AuthorizationPatchRequest)](https://github.com/camunda/camunda/blob/fc1cac538558a1a9b510888060b957bdba0f961b/zeebe/gateway-protocol/src/main/proto/rest-api.yaml#L1748) schema


The descriptions are important because the docs ingest the spec from the camunda repo, and generate our API docs from it. I'm tagging you here in the camunda-docs repo, instead of the camunda repo, because that's where we (DevEx) noticed the descriptions missing.

Please add descriptions to these endpoints/properties, in the [C8 REST OpenAPI spec](https://github.com/camunda/camunda/blob/main/zeebe/gateway-protocol/src/main/proto/rest-api.yaml). Your changes will eventually propagate downstream to the docs here.

For guidance on writing the descriptions, [see here](https://github.com/camunda/camunda/pull/21999). If you have any questions, feel free to ask. 

Thank you!